ui/src/main/webapp/src/app/components/migration-project-form.component.html

```diff
@@ -27,13 +27,13 @@
         </div>
         <div class="form-group">
             <label class="col-md-2 control-label" for="idDescription">Project Description</label>
-            <div class="col-md-6">
-                <textarea name="description" id="idDescription" class="form-control"
+            <div class="col-md-8">
+                <textarea name="description" id="idDescription" class="form-control" [rows]="Math.min(4 + (model.description ? model.description.length : 0) / 80, 25)"
                         #controlDescription="ngModel"
```

Comment:
There's an auto resizable height. Regarding width, I don't know... How long texts do you expect to end up in that field?